src/accelerate/commands/launch.py

```diff
@@ -57,6 +58,56 @@ def launch_command_parser(subparsers=None):
         action="store_true",
         help="Whether to use deepspeed.",
     )
+    parser.add_argument(
+        "--deepspeed_config_file",
+        default=None,
+        type=str,
+        help="DeepSpeed's config file (useful only when `use_deepspeed` flag is passed).",
+    )
+    parser.add_argument(
```

Comment:
Not really sure it's worth adding all of those below. The DeepSpeed config is enough IMO.